nodejs/week2/nodeJS-week2/src/backend/routes/meals.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const mealRouter = express.Router();
+const fs = require('fs');
+const mealFile = JSON.parse(fs.readFileSync(__dirname + '/../data/meals.json', 'utf-8'));
+
+mealRouter.get('/', (req, res) => {
+	res.json(mealFile);
+});
+
+mealRouter.get('/:id', (req, res) => {
+	const id = parseInt(req.params.id);
+	const mealsdetails = mealFile.filter(meal => meal.id === id);
+	mealsdetails.length > 0 && typeof id === 'number'
+		? res.send(mealsdetails)
+		: res.send('Bad request!, check if you are searching for correct mealId and its a number');
+});
+
+mealRouter.get('/api/meals', (req, res) => {
+	const maxPrice = parseInt(req.query.maxPrice);
+	const getMealSmallerThanMaxPrice = mealFile.filter(meal => meal.price < maxPrice);
```

Comment:
the get keyword suggest its a function that need to be called. But this is `mealsSmallerThanMaxprice`